cypress/integration/inventory/cataloging/catalogue-new-ordered-recieved-title.spec.js

```diff
@@ -0,0 +1,64 @@
+import newOrder from '../../../support/fragments/orders/newOrder';
+import basicOrderLine from '../../../support/fragments/orders/basicOrderLine';
```

Comment:
import fragments from the UpperCase letter

Reply:
done)